src/main_classes/Field.java

```diff
@@ -0,0 +1,44 @@
+package main_classes;
+
+import java.util.Scanner;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class Field {
+    private final int width;
+    private final int length;
+    private final String water = "\uD83C\uDF0A";
+//    private final String sand = "\uD83C\uDFDC️";
+//    private final String mountain = "⛰️";
+    private final String meadow = "\uD83D\uDFE9";
+
+    public int getWidth() {
+        return width;
+    }
+
+    public int getLength() {
+        return length;
+    }
+
+    public Field(int width, int length) {
+        this.width = width;
+        this.length = length;
+    }
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter length and width of field");
+        Field field = new Field(scanner.nextInt(), scanner.nextInt());
+        ThreadLocalRandom random = ThreadLocalRandom.current();
+
+        for (int i = 0; i < field.getWidth(); i ++){
+            for (int j = 0; j < field.getLength(); j ++){
+                int randomPicture = random.nextInt(4);
+                switch (randomPicture){
```

Comment:
switch included in for and included in another for, it's too much complexity, refactor it